contrib/localnet/QUICK_START.md

```diff
@@ -0,0 +1,86 @@
+# Quick Start Guide - Localnet with Dashboards
+
+## 🚀 Get Started in 5 Minutes
+
+### **Prerequisites:**
+- Docker Desktop installed and running
+- Git repository cloned
+
+### **Step 1: Start Services**
+```bash
+cd /Users/peterlee/go/src/github.com/zeta-chain/node/contrib/localnet
+docker compose --profile monitoring up -d
```

Comment:
I don't actually think we need this documentation, we should rather add the monitoring profile to the localnet make command here
https://github.com/zeta-chain/node/blob/a4d4fac9678559f2ade583444f7ae92d59b556eb/Makefile#L231
so its automatically spin up when using the default localnet